src/EditorFeatures/Core/Wrapping/Edit.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.PooledObjects;
+
+namespace Microsoft.CodeAnalysis.Editor.Wrapping
+{
+    /// <summary>
+    /// Represents an edit between two tokens.  Specifically, provides the new trailing trivia for
+    /// the <see cref="Edit.Left"/> token and the new leading trivia for the <see
+    /// cref="Edit.Right"/> token.
+    /// </summary>
+    internal readonly struct Edit
+    {
+        public readonly SyntaxToken Left;
+        public readonly SyntaxToken Right;
+        public readonly SyntaxTriviaList LeftTrailingTrivia;
+        public readonly SyntaxTriviaList RightLeadingTrivia;
+
+        private Edit(
+            SyntaxToken left, SyntaxTriviaList leftTrailingTrivia,
```

Comment:
See comment here: https://github.com/dotnet/roslyn/pull/30422/files/cba4acc6e64063845e0f76123dcd0a6649474c26#diff-fc04504dd0d3238e666971a22361ea66R7

This will be the *new* trailingTrivia for 'left', not the existing trivia.

Reply:
I can def add 'New' tot he name as well to make that clearer.